### PR TITLE
Add description about customizing event listener options in templates

### LINKED ISSUE #1008: Add description about customizing event listener options in templates
Customizing event listener options in templates is that an object which has `handleEvent` property is passed to event listener expressions. I think it is necessary to a description about this method to [Customizing event listener options](https://lit.dev/docs/components/events/#event-options-decorator) section.

### PR DESCRIPTION
**Pull request by 38elements**
This resolves #1008.

### CODE REVIEW

File: packages/lit-dev-content/site/docs/components/events.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ private _handleTouchStart(e) { console.log(e.type) }
 
 </div>
 
+You can customize event listener options by passing an object which has `handleEvent` property and a name whose options of `addEventListener` property to [event listener expressions](https://lit.dev/docs/templates/expressions/#event-listener-expressions):
```
Comment (arthurevans) [inline packages/lit-dev-content/site/docs/components/events.md:45]: This sentence is confusing. Perhaps something like this:

```suggestion
If you're not using decorators, you can customize event listener options by passing an object to the event listener expression. The object must have a `handleEvent()` method and can include any the options that would normally appear in the `options` argument to `addEventListener()`.
```

**Review by arthurevans — CHANGES_REQUESTED** [review]

File: packages/lit-dev-content/site/docs/components/events.md (hunk 1 above)
Comment (38elements) [reply packages/lit-dev-content/site/docs/components/events.md]: Thank you for review. I changed it.

**Review by arthurevans — APPROVED** [review]
LGTM, thanks for the fix!

**Comment by e111077** [comment]
typo in this is breaking build working on a fix-forward

**Comment by e111077** [comment]
Can't figure out why this is breaking build. Reverting for now to fix build. Will follow up monday to figure out what's going on

**Comment by 38elements** [comment]
Thank you.

**Comment by e111077** [comment]
fixed and re-added in #1020. Thanks for the submission!

**Pull request merged**